cpp/tests/rolling/range_window_bounds_test.cpp

```diff
@@ -151,5 +151,7 @@ TYPED_TEST(NumericRangeWindowBoundsTest, WrongRangeType)
                cudf::logic_error);
 }
 
+TEST_F(RangeWindowBoundsTest, Decimal) {}
```

Comment:
Should there be tests in here?

Reply:
Hmm. I'm not sure what I was thinking there. Let me take closer look.

Reply:
Added tests for `range_window_bounds` with decimal values.